CourseGrab/Home/HomeViewController.swift

```diff
@@ -10,9 +10,42 @@ import UIKit
 
 class HomeViewController: UITableViewController {
 
+    private let cellReuseId = "cell"
```

Comment:
lets make this a bit more descriptive: `private let homeCellReuseIdentifier = "homeCellReuseIdentifier"`